fdbserver/ApplyMetadataMutation.cpp

```diff
@@ -135,6 +136,7 @@ class ApplyMetadataMutationsImpl {
 	std::unordered_map<UID, StorageServerInterface>* tssMapping = nullptr;
 
 	std::map<TenantName, TenantMapEntry>* tenantMap = nullptr;
+	std::map<int64_t, TenantName>* tenantIdNameMap = nullptr;
```

Comment:
What do you think of the name `tenantIdIndex`? It's consistent with some of our other usages elsewhere, and I think suggests its relationship with the `tenantMap`.